src/plugins/explore/public/components/tabs/traces_tab.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { ExploreDataTable } from '../data_table/explore_data_table';
+import { ActionBar } from './action_bar/action_bar';
+import { setColumns } from '../../application/utils/state_management/slices';
+import { useStatusCodeFormatter } from '../formatters/use_status_code_formatter';
+
+export const TracesTab = () => {
+  const dispatch = useDispatch();
+
+  // Apply custom formatting for status codes
+  useStatusCodeFormatter();
+
+  // Set the default columns for traces when the component mounts
+  useEffect(() => {
+    dispatch(
+      setColumns([
+        'timestamp',
+        'attributes.http.status_code',
+        'status.code',
+        'attributes.rpc.method',
+        'serviceName',
+        'durationInNanos',
+      ])
+    );
+  }, [dispatch]);
```

Comment:
should we do this in the preloaded state of redux when it mounts for traces? I want to decrease the use of useEffects as much as possible in the codebase, and they are typically required when we need the react app to be in sync with some outside sync (like an api call), but this does not seem to be the case here